ocean_lib/ocean/ocean_compute.py

```diff
@@ -94,38 +80,17 @@ def status(self, did: str, job_id: str, wallet: Wallet) -> Dict[str, Any]:
         :param did: str id of the asset offering the compute service of this job
         :param job_id: str id of the compute job
         :param wallet: Wallet instance
-        :return: dict the status for an existing compute job, keys are (ok, status, statusText)
-        """
-        _, service_endpoint = self._get_service_endpoint(did)
-
-        return OceanCompute._status_from_job_info(
-            self._data_provider.compute_job_status(
-                did, job_id, service_endpoint, wallet
-            )
-        )
-
-    @enforce_types
-    def result(self, did: str, job_id: str, wallet: Wallet) -> Dict[str, Any]:
-        """
-        Gets job result.
-
-        :param did: str id of the asset offering the compute service of this job
-        :param job_id: str id of the compute job
-        :param wallet: Wallet instance
-        :return: dict the results/logs urls for an existing compute job, keys are (did, urls, logs)
+        :return: dict the status for an existing compute job
         """
         _, service_endpoint = self._get_service_endpoint(did)
-        info_dict = self._data_provider.compute_job_result(
+        job_info = self._data_provider.compute_job_status(
             did, job_id, service_endpoint, wallet
         )
-        return {
-            "did": info_dict.get("resultsDid", ""),
-            "urls": info_dict.get("resultsUrl", []),
-            "logs": info_dict.get("algorithmLogUrl", []),
-        }
+        job_info.update({"ok": job_info["status"] not in (31, 32)})
```

Comment:
I would add status code checks and use job_info.get("status") syntax to avoid errors

Reply:
The status code check is performed inside `_send_compute_request` which is called by `compute_job_status`, so it's unnecessary and impossible to repeat here because at this point we only have the `dict`, not the `Response`.

For job_info.get("status") I'm fine with that, but I'll need to add `None` to the failure conditions.  `(31, 32, None)`

P.S. As a result of this comment, I realized that I forgot to refractor the `DataServiceProvider` `compute_job_result` and `compute_job_result_file` methods the same way I did for the `OceanCompute` `result` and `result_file`.  So thank you!

Reply:
Done.  Please resolve.